From 6792d2679025d01e307b0637f63ea11e94c80435 Mon Sep 17 00:00:00 2001
From: Isaku Yamahata <isaku.yamahata@intel.com>
Date: Tue, 5 Mar 2024 15:46:14 +0800
Subject: [PATCH 01/26] Fix kernel compiling issue

Fix the compiling issue with TDX-HOST enabled in kernel config.
This fix patch is from:
https://lore.kernel.org/all/20240226192757.GS177224@ls.amr.corp.intel.com/
---
 arch/x86/include/asm/shared/tdx.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/include/asm/shared/tdx.h b/arch/x86/include/asm/shared/tdx.h
index 3e8ce567fde0..f7644423e3b4 100644
--- a/arch/x86/include/asm/shared/tdx.h
+++ b/arch/x86/include/asm/shared/tdx.h
@@ -27,6 +27,8 @@
 #define TDVMCALL_GET_QUOTE		0x10002
 #define TDVMCALL_REPORT_FATAL_ERROR	0x10003
 
+#define TDVMCALL_STATUS_RETRY          1
+
 /*
  * TDG.VP.VMCALL Status Codes (returned in R10)
  */
-- 
2.34.1

